docs/program/apis/sessions.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Session Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the session management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The session management API provides support for robot session management.
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### Purpose of session management
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server`.
+
+The period of time during which these clients are present is called a *session*.
+*Session management* is a presence mechanism between a client and `viam-server`.
+
+Session management allows for safer operation of robots that physically move.
+For example, imagine a wheeled rover gets a [`SetPower()`](/components/base/#setpower) command as the last input from a client before the connection to the robot is interrupted.
+Without session management, the API request from the client sets the flow of electricity to the motors of the robot and then does not time out, causing the robot to continue driving forever, colliding with objects and people.
+
+With session management, if the robot does not receive a signal from the client at regular intervals, it will safely stop until the connection is reestablished.
+
+#### Clients and Viam's client SDKs
```

Comment:
I find this title kinda odd because a client SDK can act as a client right? So it's kinda like "Rectangles and squares"? And then it goes on to talk about heartbeats too?

What would you think of titling this section "How session management works" or "Clients and heartbeats" or something?

Reply:
re. this I added a `heartbeats` section~~ see thread closer to top for more ref. info!